src/app/components/Transactions/ConsensusTransactions.tsx

```diff
@@ -0,0 +1,148 @@
+    { key: 'block', content: t('common.block') },
+    { key: 'age', content: t('common.age'), align: TableCellAlign.Right },
+    ...(verbose
+      ? [
+          { key: 'type', content: t('common.type'), align: TableCellAlign.Center },
+          { key: 'from', content: t('common.from'), width: '150px' },
+          { key: 'to', content: t('common.to'), width: '150px' },
+          { key: 'txnFee', content: t('common.transactionFee'), align: TableCellAlign.Right, width: '250px' },
+          { key: 'value', align: TableCellAlign.Right, content: t('common.value'), width: '250px' },
+        ]
+      : []),
+  ]
+  const tableRows = transactions?.map(transaction => ({
+    key: transaction.hash,
+    data: [
+      {
+        content: <StatusIcon success={transaction.success} error={transaction.error} />,
+        key: 'success',
+      },
+      {
+        content: <TransactionLink scope={transaction} alwaysTrim={true} hash={transaction.hash} />,
+        key: 'hash',
+      },
+      {
+        content: <BlockLink scope={transaction} height={transaction.block} />,
+        key: 'round',
+      },
+      {
+        align: TableCellAlign.Right,
+        content: <Age sinceTimestamp={transaction.timestamp} />,
+        key: 'timestamp',
+      },
+      ...(verbose
+        ? [
+            {
+              align: TableCellAlign.Center,
+              // TODO: needs icons for ConsensusTxMethod
+              content: <>-</>,
+              key: 'type',
+            },
+            {
+              align: TableCellAlign.Right,
+              content: (
+                <Box
+                  sx={{
+                    display: 'flex',
+                    alignItems: 'center',
+                    position: 'relative',
+                    pr: 3,
+                  }}
+                >
+                  {!!ownAddress && transaction.sender === ownAddress ? (
+                    <Typography
+                      variant="mono"
+                      component="span"
+                      sx={{
+                        fontWeight: 700,
+                      }}
+                    >
+                      {trimLongString(transaction.sender)}
+                    </Typography>
+                  ) : (
+                    <AccountLink scope={transaction} address={transaction.sender} alwaysTrim={true} />
+                  )}
+                </Box>
+              ),
+              key: 'from',
+            },
+            {
+              // TODO: show recipients address when props is added to API
+              content: <>-</>,
+              key: 'to',
+            },
+            {
+              align: TableCellAlign.Right,
+              content: <RoundedBalance value={transaction.fee} ticker={transaction.ticker} />,
+              key: 'fee_amount',
+            },
+            {
+              align: TableCellAlign.Right,
+              // TODO: show RoundedBalance when API returns amount prop
```

Comment:
So, both the amount and the receiver of the TX is missing from the API? 😞

Reply:
There is backend ticket to fix that https://app.clickup.com/t/869353k3r

Reply:
note for myself: update API layer when props are added (fromBaseUnits) etc